spec/values/calculation/calc/error/known_incompatible/minus.hrx

```diff
@@ -0,0 +1,10 @@
+<===> input.scss
```

Comment:
Is this file a duplicate of `spec/values/calculation/calc/error/known_incompatible/operators.hrx` ?